README.md

```diff
@@ -5,12 +5,20 @@ Reaction Database.
 
 ## Getting Started
 
-From the ord-schema project root, 
+Before using ord-editor, you must first clone
+[https://github.com/Open-Reaction-Database/ord-schema/](ord-schema) also.
```

Comment:
Cloning ord-schema is only required if you are not using the dockerfile to build the editor...could you please make this clear?

Reply:
Thanks! I misunderstood.